app/src/main/res/css/modules/item_card.sass

```diff
@@ -41,7 +40,7 @@
     width: 100%
 .item_card.shadow
   &:hover
-    box-shadow: 0 10px 20px rgba(0,0,0,0.19), 0 6px 6px rgba(0,0,0,0.23)
+    box-shadow: 0 2px 0px rgba(0, 0, 0, 0.19), 0 3px 4px rgba(0, 0, 0, 0.23)
```

Comment:
@tahnik, OK, let's keep the previous one, but please make sure it has enough `padding`.